### PR TITLE
chore: add utils package

### PR DESCRIPTION
**Pull request by proquig**
## Description

This adds a utils package that contains a collection of utils. For now it only contains crypto utils.

## Checklist:

Before you create this PR, confirm all the requirements listed below by checking the checkboxes like this (`[x]`).

- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests to cover the new feature or fixes.

### CODE REVIEW

File: packages/utils/src/crypto.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  const bytes = Uint8Array.from(matches.map((hex) => parseInt(hex, 16)));
+  return bytes.buffer;
+};
+
+const hexStringFromArrayBuffer = (buffer: ArrayBuffer) => {
+  return [...new Uint8Array(buffer)].map((byte) => byte.toString(16).padStart(2, '0')).join('');
+};
+
+export const encryptAES256GCM = async ({ data, keyHex }: { data: string; keyHex: string }) => {
```
Comment (elbalexandre) [inline packages/utils/src/crypto.ts:15]: minor, but IMO `encryptAES256GCM(data: string, key: string)` is in this case a better signature for callee and declaration.

Do we really need to precise `keyHex` ? looks odd as `userIdUuid`.

**Review by elbalexandre — COMMENTED** [review]
Can we add test with vitest edge-runtime env ?
We might want to rename `encryptAES256GCM` & `decryptAES256GCM`. IMO it's a bit too barbaric. Perhaps `encryptText` & `decryptText` could be fine.

**Review by elbalexandre — APPROVED** [review]
